generator/generator.go

```diff
@@ -0,0 +1,148 @@
+// +build ignore
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	_ "github.com/docker/machine/drivers/amazonec2"
+	//	_ "github.com/docker/machine/drivers/azure"
+	_ "github.com/docker/machine/drivers/digitalocean"
+	_ "github.com/docker/machine/drivers/exoscale"
+	//	_ "github.com/docker/machine/drivers/google"
```

Comment:
We are not doing google and azure now, because they require some UI and cattle changes